standard_reports/item-usage/latest/report-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "is_custom": false,
-  "version": "2.6.1",
+  "version": "2.6.2",
```

Comment:
not .3 ?

Reply:
Nope, report don't follow app versioning they just get bumped up patch when they changed, or if there is backward compatibility break they get bumped up version to the minor of omSupply and previous version is split for backwards compatibility:

https://github.com/msupply-foundation/open-msupply/tree/develop/standard_reports#report-versioning